pkgs/by-name/gh/ghostscript/package.nix

```diff
@@ -96,6 +96,8 @@ stdenv.mkDerivation rec {
   ;
 
   preConfigure = ''
+    export DARWIN_LDFLAGS_SO_PREFIX=$out/lib/
```

Comment:
We could probably do this when building for Darwin only, except I wasn't sure how to do that. 😅 It doesn't do anything on non-Darwin anyway, so it currently doesn't hurt.